app/services/food_service.rb

```diff
@@ -0,0 +1,7 @@
+class FoodService
+    def self.search(keyword)
+        conn = Faraday.new(url: 'https://api.nal.usda.gov/fdc/v1/foods')
+        response = conn.get("/search?api_key=#{ENV['food_api_key']}&query=#{keyword}")
+        JSON.parse(response.body, symbolize_names: true)
+    end
```

Comment:
When you think about your connection, consider using just the domain. The endpoint is `/fdc/v1/foods/search`, so that's what you can have in your get method. This is me just being a little picky ;)